concent_api/core/message_handlers.py

```diff
@@ -1019,6 +1025,82 @@ def store_message(
     return stored_message
 
 
+def handle_send_subtask_results_verify(
+    request,
+    subtask_results_verify: message.concents.SubtaskResultsVerify
+):
+    subtask_results_rejected = subtask_results_verify.subtask_results_rejected
+    report_computed_task = subtask_results_rejected.report_computed_task
+    compute_task_def = report_computed_task.task_to_compute.compute_task_def
+    validate_id_value(
```

Comment:
Wouldn't it be better to use validation for task_to_compute, which already has this check plus few more ?

Reply:
it was there, I must have lost it during rebase :/, will fix

Reply:
EDIT: Actually I forgot I have refactoed it and now `validate_golem_message_subtask_results_rejected` does it inside